scallop_model_fun.R

```diff
@@ -192,9 +191,11 @@ scallop_model_fun <- function(scenario){
             qt[i] <- scenario$catch$qmax/(1+scenario$per.rec$kq*B[i-1])                                                    #q as a function of vul bio
           }
           et[i] <- scenario$catch$effort[timer[i]] * scenario$catch$e_years[yr.timer[i]] #Calculation of Effort
-          hr[i] <- 1-exp(-qt[i]*et[i])
-          hr_sel[i,] <- hr[i]*scenario$life.vec$Vul
-          hr_harv[i,] <- hr[i]*scenario$life.vec$Vul*scenario$catch$p.legal
+          # hr[i] <- 1-exp(-qt[i]*et[i])
+          # hr_sel[i,] <- hr[i]*scenario$life.vec$Vul
+          hr_sel[i,] <- 1-exp(-qt[i]*et[i]*scenario$life.vec$Vul)
+          # hr_harv[i,] <- hr[i]*scenario$life.vec$Vul*scenario$catch$p.legal
+          hr_harv[i,] <- hr_sel[i,]*scenario$catch$p.legal
```

Comment:
p.legal is in there for a size-limit from Ed's initial port of the bag limit code. We can remove.